index.html

```diff
@@ -1,56 +1,65 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
-    <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <link href='https://fonts.googleapis.com/css?family=Poppins' rel='stylesheet'>
-    <link rel="stylesheet" href="./style.css" type="text/css">
-    <title>Portfolio Homepage</title>
+  <meta charset="UTF-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <link href="https://fonts.googleapis.com/css?family=Poppins" rel="stylesheet" />
+  <link rel="stylesheet" href="./style.css" type="text/css" />
+  <title>Portfolio Homepage</title>
 </head>
+
 <body>
-    
   <!-- Navigation Bar Section -->
 
   <div class="wrapper">
     <div class="front-wrapper">
       <header>
-        <nav class="flex-nav">
-          <a href="https://yussufob.github.io/My-Portfolio-Part-II/" class="my-logo">Y-Bash</a>
-          <a href="https://yussufob.github.io/My-Portfolio-Part-II/" class="union-icon"><img src="./images/Union.svg" alt="list-icons"></a>
-          <ul class="menu-list">
-            <li><a href="https://yussufob.github.io/My-Portfolio-Part-II/">Portfolio</a></li>
-            <li><a href="https://yussufob.github.io/My-Portfolio-Part-II/">About</a></li>
-            <li>Contact</li>
+        <nav>
+          <div class="logo">Bash</div>
+          <div class="openMenu"><img src="./images/Union.svg" alt="hamburger"></div>
+          <ul class="mainMenu">
+            <li class="shieldOff1"><a href="#my-portfolio">Portfolio</a></li>
+            <li class="shieldOff2"><a href="#about-myself">About</a></li>
+            <li class="shieldOff3"><a href="#contact-me">Contact</a></li>
+            <li class="closeMenu"><img  src="./images/Close_Icon.svg" alt="closeMenu"></li>
```

Comment:
You must stick to the design as much as possible (margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).

Your Implementation | Figma Design
:---:|:---:
![image](https://user-images.githubusercontent.com/10332499/161719527-b1beb258-6f6b-4f4c-97fb-a8ea1cb9ffa8.png) | ![image](https://user-images.githubusercontent.com/10332499/161719431-57f0ef51-4a5a-4569-a55b-970084b55ac7.png)